pyccel/codegen/printing/ccode.py

```diff
@@ -685,6 +685,8 @@ def _print_Rand(self, expr):
     def _print_NumpyRandint(self, expr):
         raise NotImplementedError("Randint not implemented")
 
+    def _print_Interface(self, expr):
```

Comment:
is this printer not supposed to call the printer for every function in the interface?

Reply:
It doesn't, it only prints the interfaces, and since there are none in this case, it returns an empty string